index.js

```diff
@@ -8,7 +8,9 @@ const run = async () => {
 }
 
 try {
-  run()
+  run().catch((error) => {
```

Comment:
since run() is async, we can omit the try catch block and just do:

```
 run().catch((error) => {
    core.setFailed(`Action failed with error: ${error}`)
  })
```

Thanks for doing this!